Kickstarter-iOS/Views/Controllers/ProjectPageViewControllerTests.swift

```diff
@@ -590,6 +590,15 @@ internal final class ProjectPageViewControllerTests: TestCase {
       fetchProjectPamphletResult: .failure(.couldNotParseJSON)
     )
 
+    // This test was previously flakey on CI because it relied on Alamofire to download an image url to populate the user image, which may/may not be ready in time.
```

Comment:
Yea that's actually a bug, Alamofire does have the functionality to add a placeholder that isn't being used right now. I'm hesitant to change that because it's an application change that will affect that page rendering. Very small and probably an improvement, but I just want to quickly fix the tests here to suit the existing application logic, reason being is because it needs further testing and some tests may need re-recording.